tsconfig.json

```diff
@@ -66,5 +66,6 @@
         /* Experimental Options */
         // "experimentalDecorators": true,        /* Enables experimental support for ES7 decorators. */
         // "emitDecoratorMetadata": true,         /* Enables experimental support for emitting type metadata for decorators. */
-    }
+    },
+    "exclude": ["cypress"]
```

Comment:
What's the reason for adding this here?

Reply:
good question, in order to get the `tab`action to work in cypress we had to add this experimental packages: `cypress-plugin-tab`. However after adding it I started getting type issues on all the unit tests in the repo. After trying out several solutions, I found out that getting TS to ignore the cypress folder fixed the problem. As cypress does not support TS, i think it should not affect anything anyway.